ci-scripts/release.sh

```diff
@@ -0,0 +1,8 @@
+
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+IMAGE="teletracking/rds_exporter"
+
+docker push "${IMAGE}:latest"
```

Comment:
@sandesh9989 Let's get this versioned. Take a look at the SAST-Scanner for refernce.

Reply:
Done